### DISCUSSION TITLE
MenuMeters on the bar

### DISCUSSION THREAD

**Discussion by axonasif** [FelixKratz/SketchyBar#179] [Q&A] [answered]
Hello!
Is it possible to render things like MenuMeters on the bar?

The reason I want to use it is that I feel it's more convenient and native than scripting with a bunch of external CLI tools to extract data from (which is kind of a mess for this particular case TBH).

**Comment by FelixKratz** [comment, marked as answer]
Yes that is possible, look for the „alias“ in the Readme. This way you can display *all* regular menu bar apps in sketchybar, while being able to modify their properties as ypu can with all native skechtybar items.
